skyportal/handlers/api/random_string.py

```diff
@@ -0,0 +1,25 @@
+import uuid
```

Comment:
This is kind of a weird thing to have hanging around SkyPortal.  Any thoughts on how we can test it without adding it to the "official" application?